terraform/parameter_store.tf

```diff
@@ -2,26 +2,34 @@ resource "aws_ssm_parameter" "wordpress-db-host" {
   name        = "/wordpress/WORDPRESS_DB_HOST"
   description = "The host parameter to be used by the container and DB"
   type        = "SecureString"
-  value       = "secret"
+  value       = random_password.value.result
 }
 
 resource "aws_ssm_parameter" "wordpress-db-user" {
   name        = "/wordpress/WORDPRESS_DB_USER"
   description = "The user parameter to be used by the container and DB"
   type        = "SecureString"
-  value       = "secret"
+  value       = random_password.value.result
 }
 
 resource "aws_ssm_parameter" "wordpress-db-password" {
   name        = "/wordpress/WORDPRESS_DB_PASSWORD"
   description = "The password parameter to be used by the container and DB"
   type        = "SecureString"
-  value       = "secret"
+  value       = random_password.value.result
 }
 
 resource "aws_ssm_parameter" "wordpress-db-name" {
   name        = "/wordpress/WORDPRESS_DB_NAME"
   description = "The name parameter to be used by the container and DB"
   type        = "SecureString"
-  value       = "secret"
+  value       = random_password.value.result
```

Comment:
you should use the value that coorespond to the secret.. in that case the db-name